data/event-schemas/schema/m.room_key.withheld.yaml

```diff
@@ -48,8 +48,8 @@ properties:
       session_id:
         type: string
         description: |-
-          Required of `code` is not `m.no_olm`.  The session ID of the key
-          that this event is aboutis for.
+          Required if `code` is not `m.no_olm`.  The session ID of the key
+          that this event is about.
```

Comment:
"Is about" seems weirdly phrased, but I won't block the PR on it.

Reply:
Yeah.  The other properties also say "is about", so I kept it for consistency.